"""Generated protobuf bindings for SCIP (Sourcegraph Code Intelligence Protocol).

See: https://github.com/sourcegraph/scip
"""

from contextmine_core.semantic_snapshot.proto.scip_pb2 import (
    Descriptor,
    Diagnostic,
    DiagnosticTag,
    Document,
    Index,
    Language,
    Metadata,
    Occurrence,
    Package,
    PositionEncoding,
    ProtocolVersion,
    Relationship,
    Severity,
    Symbol,
    SymbolInformation,
    SymbolRole,
    SyntaxKind,
    TextEncoding,
    ToolInfo,
)

__all__ = [
    "Descriptor",
    "Diagnostic",
    "DiagnosticTag",
    "Document",
    "Index",
    "Language",
    "Metadata",
    "Occurrence",
    "Package",
    "PositionEncoding",
    "ProtocolVersion",
    "Relationship",
    "Severity",
    "Symbol",
    "SymbolInformation",
    "SymbolRole",
    "SyntaxKind",
    "TextEncoding",
    "ToolInfo",
]
